tasks/main.yml

```
@@ -717,10 +717,7 @@
         adutil user create
         --name {{ mssql_ad_sql_user_name }}
         --password {{ mssql_ad_sql_password | quote }}
-        --distname
-        CN={{ mssql_ad_sql_user_name }},CN=Users,DC={{
-        ad_integration_realm.split('.') | first | upper
-        }},DC=COM
+        --distname {{ mssql_ad_sql_user_dn | regex_replace(' ', '') }}
```

```suggestion
        --distname {{ mssql_ad_sql_user_dn | quote }}
```
A distinguished name value can have a lot of shell metacharacters in it